local_planner/src/nodes/planner_functions.cpp

```diff
@@ -63,8 +63,7 @@ void processPointcloud(pcl::PointCloud<pcl::PointXYZI>& final_cloud,
 
         // only remember point if it's in a cell not previously populated by
         // complete_cloud, as well as outside FOV and 'young' enough
-        if (histogram_points_counter(p_ind.y(), p_ind.x()) < min_num_points_per_cell && xyzi.intensity < max_age &&
```

Comment:
We could try `histogram_points_counter(p_ind.y(), p_ind.x()) < 4 * min_num_points_per_cell`

Reply:
I think a better way would be to make the resolution of the downsampling grid 4x4 higher than the histogram grid, rather than 2x2 higher. This would allow points to get to 60 degrees off-center before any cells in the lower resolution grid would be empty upon return.